### ISSUE TITLE
Different type of entry - no years

### ISSUE THREAD

**Issue by davidmihalyi** [nstrayer/datadrivencv#12]
It would be nice to have entries on my CV without any year variable attached to them, like a running list of conferences or names of references. Is there a quick tweak for that?

**Comment by nstrayer** [comment]
Definitely possible (and should be in the product already). To add it just replace [this section in `CV_printing_functions.R`](https://github.com/nstrayer/datadrivencv/blob/master/inst/templates/CV_printing_functions.R#L67-L74) with 

```r
dplyr::mutate(
      description_bullets = paste0("- ", description_bullets),
      no_start = is.na(start),
      has_start = !no_start,
      no_end = is.na(end),
      has_end = !no_end,
      cur_end = tolower(end) %in% c("current", "now", ""),
      end_num = ifelse (cur_end | no_end, future_year, end),
      timeline = dplyr::case_when(
        no_start  & no_end  ~ "N/A",
        no_start  & has_end ~ as.character(end),
        has_start & no_end  ~ paste(start, "-", "Current"),
        TRUE                ~ paste(end, "-", start)
      )
    ) %>%
    dplyr::select(-no_start, -has_start, -no_end, -has_end, -cur_end)
```

Now if you omit a start or end date nothing will show up (but it will be sorted to the top of a section). 
Note that this will break the network visualization in its current form because it relies on a date range for its connections (but I will update it shortly)

**Comment by davidmihalyi** [comment]
This worked great! 
Just flagging that you should also add another %>% after the last select command when replacing section shown.
d

**Comment by nstrayer** [comment]
Wonderful. Thanks! The dangers of copying and pasting code

**Comment by lepardov2020** [comment]
I also want to put a neat reference section at the end. At the beginning, I tried with the "text_blocks" but they don´t look nice. So I followed this thread and pasted the bit of code, but I still can´t have the date cell empty. Any thoughts? once I put a date, I have a nice reference section but would like to delete the date in the left hand. (I even started over from the beginning, installing the tools and everything, but it wouldn´t let me and I received a message saying that nothing has changed and if I wanted to force the installation, I said no...)
